### PR TITLE
[ZT] DNS scheduled policies

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/dns-policies/scheduled-dns-policies.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+pcx_content_type: reference
+title: Scheduled DNS policies
+weight: 3
+---
+
+# Scheduled DNS policies
+
+Cloudflare Gateway allows you to configure any DNS policy to activate or deactivate on a regular time interval. 
+
+By default, Gateway policies are unscheduled and apply at all times. You can use the [Gateway Rules API](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule) to create a scheduled DNS policy or add a schedule to an existing policy. To schedule a policy, send a [`POST`](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule) or [`PUT`](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-update-zero-trust-gateway-rule) request with the `schedule` parameter set to your desired days of the week, times of day, and time zone. The schedule will appear in the Zero Trust dashboard under **Gateway** > **Policies** > **DNS** when you expand the row for the policy.
```
Comment (Encore-Encore) [inline content/cloudflare-one/policies/filtering/dns-policies/scheduled-dns-policies.md]: time zone is optional

Comment (ranbel) [inline content/cloudflare-one/policies/filtering/dns-policies/scheduled-dns-policies.md]:
```suggestion
By default, Gateway policies are unscheduled and apply at all times. You can use the [Gateway Rules API](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule) to create a new DNS policy with a schedule or add a schedule to an existing policy. To schedule a policy, send a [`POST`](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule) or [`PUT`](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-update-zero-trust-gateway-rule) request with the `schedule` parameter set to your desired days of the week, times of day, and time zone. The schedule will appear in the Zero Trust dashboard under **Gateway** > **Policies** > **DNS** when you expand the row for the policy.
```

File: content/cloudflare-one/policies/filtering/dns-policies/scheduled-dns-policies.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        "traffic": "dns.fqdn == \"clockin.com\"",
+        "enabled": true,
+        "schedule": {
+            "sat":  "00:00-24:00",
+            "sun":  "00:00-24:00"
+        }
+    }'
+```
+
+The user's time zone is inferred from their source IP address. If Gateway is unable to determine the time zone from the IP, we will fall back to the time zone of the data center where the query was received.
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/dns-policies/scheduled-dns-policies.md]:
```suggestion
The user's time zone is inferred from the IP geolocation of their source IP address. If Gateway is unable to determine the time zone from the IP, we will fall back to the time zone of the data center where the query was received.
```

Comment (ranbel) [reply content/cloudflare-one/policies/filtering/dns-policies/scheduled-dns-policies.md]: move this info to its own section above and explain the Fixed Timezone logic.

**Review by patriciasantaana — APPROVED** [review]

**Pull request merged**